helm/spark-yotpo/templates/sparkapplication.yaml

```
@@ -0,0 +1,161 @@
+apiVersion: sparkoperator.k8s.io/v1beta2
+kind: SparkApplication
+metadata:
+  name:  {{ .Release.Name | trunc 63 }}
+  labels:
+    chartname: {{ .Chart.Name | trunc 63 | quote }}
+    release: {{ .Release.Name | trunc 63 | quote }}
+    revision: {{ .Release.Revision | quote }}
+    sparkVersion: {{ .Values.sparkVersion | quote }}
+    version: {{ .Chart.Version | quote }}
+spec:
+  type: Scala
+  mode: cluster
+  image: {{ list .Values.imageRegistry .Values.image | join "/" | quote }}
```

```suggestion
  image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```
this is the best practice way of doing it. Default can change to w/e we want. Probably should be something like "master"